packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalConfirmationDialog.tsx

```diff
@@ -159,9 +159,9 @@ export function WithdrawalConfirmationDialog(props: UseDialogProps) {
   const [checkbox2Checked, setCheckbox2Checked] = useState(false)
 
   const bothCheckboxesChecked = checkbox1Checked && checkbox2Checked
+  const confirmationDays = Math.floor(l1.network.blockTime * l2.network.confirmPeriodBlocks / 86400)
```

Comment:
would be good to swap the magic num `86400 ` for a const with a name explaining it